private/buf/cmd/buf/buf_test.go

```diff
@@ -2279,7 +2279,7 @@ func TestConvertRoundTrip(t *testing.T) {
 	)
 	t.Run("stdin and stdout", func(t *testing.T) {
 		t.Parallel()
-		stdin := bytes.NewBuffer([]byte(`{"one":"55"}`))
+		stdin := bytes.NewBufferString(`{"one":"55"}`)
```

Comment:
The is the new `mirror` linter that recommends appropriate functions if ones exist for both string/[]byte.